pipeline/e2e-test.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+steps:
+    # e2e tests depend drop folder. run build:all will create said folder
```

Comment:
I think including the 2 "publish" tasks for the e2e results/screenshots is at least as reasonable as including build:all here :)

Reply:
I though about that... in the long run though, if we are able to run unit/e2e test on ubuntu, would we want to publish test results from that run too??? So I didn't move it, it should be a problem we solve in the future.

Reply:
I think we definitely would want to publish test results from both places; we'd need to in order to debug failures effectively.

Reply:
I prefer we make those changes when we get there, basically YAGNI